sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/V1WriteCommandSuite.scala

```diff
@@ -181,13 +178,111 @@ class V1WriteCommandSuite extends QueryTest with SharedSparkSession with V1Write
             |PARTITIONED BY (k STRING)
             |""".stripMargin)
         executeAndCheckOrdering(
-          hasLogicalSort = true, orderingMatched = enabled, hasEmpty2Null = enabled) {
+          hasLogicalSort = true, orderingMatched = true, hasEmpty2Null = enabled) {
```

Comment:
do we still need the `orderingMatched` parameter if it's always true?

Reply:
Not sure what you mean, as in needed in this test?

Reply:
I mean, shall we remove `orderingMatched` from the method `executeAndCheckOrdering`?

Reply:
There is no default value for `orderingMatched` and two other unit tests still use `orderingMatched=enabled`.